scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -81,64 +80,6 @@ do
     esac
 done
 
-params=$@
-
-_install_prerequisites_centos()
-{
-    # yum doesn't accept timeout in seconds as parameter
-    echo
-    echo "Note: if yum becomes non-responsive, try aborting the script and run:"
-    echo "      sudo -E $0"
-    echo
-
-    CMDS=("yum -y install tar libpciaccess numactl-libs"
-          "yum -y groupinstall 'Development Tools'"
-          "yum -y install rpmdevtools openssl openssl-devel bc numactl ocl-icd ocl-icd-devel")
-
-    for cmd in "${CMDS[@]}"; do
-        echo "$cmd"
-        eval "$cmd"
-        if [[ $? -ne 0 ]]; then
-            echo "ERROR: failed to run $cmd" >&2
-            echo "Problem (or disk space)?" >&2
-            echo ". Verify that you have enough disk space, and run the script again." >&2
-            exit $EXIT_FAILURE
-        fi
-    done
-
-}
-
-_install_prerequisites_ubuntu()
-{
-    CMDS=("apt-get -y update"
-          "apt-get -y install libnuma1 ocl-icd-libopencl1")
```

Comment:
Please revert ocl-icd-libopencl1, without it we get missing libOpenCL.so.1 lib. The same issue for CentOS

Reply:
This package will be installed automatically with runtime:
```
apt list -i *opencl*
Listing... Done
intel-igc-opencl/now 1.0.4756 amd64 [installed,local]
intel-opencl/now 20.35.17767 amd64 [installed,local]
ocl-icd-libopencl1/bionic,now 2.2.11-1ubuntu1 amd64 [installed,automatic]
```

Reply:
Uh, I was wrong, it was installed with _clinfo_ in my case. I suggest adding `ocl-icd-libopencl1 lsb-release wget` since these tools are required by the script.

Reply:
Done